### PR TITLE
GLT-2128: improve performance of list pages

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Dtos.java#L432](https://github.com/TGAC/miso-lims/blob/143ec90b95b154e000ba83060cd9dc74e57a4e2d/miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java#L432): Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java [hunk 1]
```diff
@@ -1052,7 +1052,7 @@ public int compare(SequencingParametersDto o1, SequencingParametersDto o2) {
     return dtoList;
   }
 
-  public static LibraryDto asDto(Library from) {
+  private static LibraryDto asDto(Library from, boolean initializeBoxables) {
```
Comment (callunity) [inline miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: `includeX` seems to be idiomatic elsewhere in the file, as well as in the Sample method above; what do you think about changing `initializeBoxables` to `includeBoxables` here and in the Dilution asDto method below?

Comment (djcooke) [reply miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: changed

**Comment by sqwci** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [Dtos.java#L432](https://github.com/TGAC/miso-lims/blob/b4875d23eddfad6a3f61371e4d66be5d0483e132/miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java#L432): Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

**Pull request merged**